lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -263,7 +264,7 @@ def _get_key(self, event):
         for key_mask, prefix in modifiers:
             if event.state & key_mask:
                 if not (prefix == 'shift' and unikey.isprintable()):
-                    key = '{0}+{1}'.format(prefix, key)
+                    key = f'{prefix}+{key}'
```

Comment:
@tacaswell says there's good reason code cove is cranky, and none of these are actually tested - does that mean it needs tests?

Reply:
We test key press events in Matplotlib only. This code is used when processing key events from GTK, to translate to Matplotlib. @anntzer tried to test this end, but the fake event creator in GTK doesn't work.

Reply:
@story645 Are you meaning to block on this?

Reply:
No but I'm not sure what the resolution is